.travis.yml

```diff
@@ -19,7 +19,7 @@ before_install:
   - openssl aes-256-cbc -K "$encrypted_5ebd3ff04788_key" -iv "$encrypted_5ebd3ff04788_iv" -in src/bin/travis/resources/jesConf.tar.enc -out jesConf.tar -d || true
 env:
   global:
-    - CENTAUR_BRANCH=develop
+    - CENTAUR_BRANCH=cjl_cromwell-2094
```

Comment:
Remember to revert before merging.
Will need to review and merge https://github.com/broadinstitute/centaur/pull/195 at the same time.